docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
+own configuration. This doesn't prevent you from granting permissions to those
+projects site-wide by altering default groups or creating new one, or creating
+additional custom settings for individual component as described bellow.
 
-.. _manage-users:
+One of the main benefits of managing the permissions through the Weblate
+interface is that you can delegate it to other users without giving them
+superuser privileges. In order to do so just add them to the `Administration`
+group of the project.
 
-Managing users and groups
--------------------------
+Custom permissions for languages, components or projects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-All users and the various groups they are in can be managed using the
-Django admin interface available, which you can get to by appending
-:file:`/admin/` to the Weblate site URL.
+You can create your own dedicated groups to manage permissions for distinct
+objects such as languages, components and projects. Although this groups can
+only grant additional privileges, you are not able to revoke any permission
+granted by site-wide or per-project groups by adding a yet another custom group.
 
-.. _manage-acl:
+**Example:**
 
-Managing per-project access control
-+++++++++++++++++++++++++++++++++++
+  If you want (for whatever reason) to allow translation to a
+  specific language (lets say `Czech`) only to a closed set of reliable users
+  while keeping translations to other languages public, you will have to:
 
-.. note::
+  1. Remove the permission to translate to the `Czech` from all the users. In the
+     default configuration this can be done by altering the `Users`
+     :ref:`default group <default-groups>`.
 
-    This feature only works for projects using access control, see :ref:`acl`.
+     .. list-table:: Group `Users`
+         :stub-columns: 1
 
-Users with the :guilabel:`Manage project access` privilege (see
-:ref:`privileges`) can also manage users in projects with access control
-turned on through the project page. The interface allows you to:
+         * - Language selection
+           - `As defined`
+         * - Languages
+           - All but `Czech`
 
-* Add existing users to the project
-* Invite new users to the project
-* Change user permissions
-* Revoke user access
+..
 
-.. versionadded:: 3.11
+  2. Add a dedicated group for `Czech` translators.
 
-* Resend the e-mail for user invitations (invalidating any previously sent invitation)
+     .. list-table:: Group `Czech translators`
+         :stub-columns: 1
 
-User management is available in the :guilabel:`Manage` menu of any project:
+         * - Roles
+           - `Power users`
+         * - Project selection
+           - `All public projects`
+         * - Language selection
+           - `As defined`
+         * - Languages
+           - `Czech`
 
-.. image:: /images/manage-users.png
+..
 
-.. seealso::
+  3. Add to the group some users you wish to give the permissions to.
```

Comment:
```suggestion
  3. Include in the group some users you wish to grant the permissions to.
```
Guessing the way to do it is by way of configuring the group, and that is why this isn't "Add some users to the group to grant them these permissions"

Reply:
thanks for the point, rephrased.